cursor.go

```diff
@@ -44,6 +44,17 @@ func (c *Cursor) Select(fields ...string) *Cursor {
 	return c
 }
 
+// GetStripe retrieves the stream information for the specified stripe.
+func (c *Cursor) GetStripe(n int) error {
```

Comment:
This isn't really Getting a stripe. It's skipping ahead to one. Not sure what the right verb to use in the context of this library is... Perhaps `SkipToStripe` or something?

Reply:
You're right, we're just moving forward to the stripe in question. I was thinking about `SelectStripe(n)` but `SkipToStripe` makes sense as well.

Reply:
`SelectStripe` sounds good to me.